packages/flutter/test/widgets/shortcuts_test.dart

```diff
@@ -106,10 +202,73 @@ void main() {
               'keys: {LogicalKeyboardKey#00000(keyId: "0x00000061", keyLabel: "a", debugName: "Key A"), LogicalKeyboardKey#00000(keyId: "0x00000062", keyLabel: "b", debugName: "Key B")}'));
     });
   });
-  group(ShortcutManager, () {
-    test('$ShortcutManager .', () {
-
+  group(Shortcuts, () {
```

Comment:
D'oh! Sorry I missed this myself in the last review.

Reply:
No worries.  Nobody was using it yet anyhow.